software/yamtrack.yml

```
@@ -0,0 +1,13 @@
+name: "Yamtrack"
+website_url: "https://github.com/FuzzyGrim/Yamtrack"
+source_code_url: "https://github.com/FuzzyGrim/Yamtrack"
+description: "Media tracker for movies, tv shows, anime, manga, video games and books."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - Python
+tags:
+  - Miscellaneous
```

Since its streaming tools, I think it should be here?

```suggestion
  - Media Streaming - Multimedia Streaming
```

---

You can't actually stream anything with this. It is only there to track your progress on series, mangas, etc.